### DISCUSSION TITLE
LIS Noah-MP3.6 not giving PET output separately

### DISCUSSION THREAD

**Discussion by Arijit22IITDelhi** [NASA-LIS/LISF#1520] [Q&A]
Hello, I was running Noah-MP V3.6 within LIS V7.4. I enabled the Potential Evapotranspiration option in the output attributes table, but it is showing an error while running with this output specification. What necessary changes are required in the LIS config file and input file to get potential evapotranspiration in output?

**Comment by dmocko** [comment]
Hi @Arijit22IITDelhi 

The Noah-MP-3.6 LSM does not calculate PET, thus this variable is not at all available as an output.

That said, we have tried to add our own calculation of PET in Noah-MP-4.0.1, if you wanted to use that version of the LSM instead.

**Reply by Arijit22IITDelhi** [reply]
@dmocko Thank you so much. One more query is that in order to get LAI, GPP, NEE etc in output I need to turn on the dynamic vegetation option. So out of 9 options which one is preferred  for " Noah-MP.4.0.1 dynamic vegetation option:" ?

**Reply by dmocko** [reply]
Hi - please see the LIS Users' Guide.  We indicate the preferred options for Noah-MP in there.

https://nasa-lis.github.io/LISF/LIS_users_guide/LIS_users_guide.html

For Noah-MP-4.0.1, to use dynamic vegetation (really, it's prognostic phenology), the recommended option is "5".

**Reply by Arijit22IITDelhi** [reply]
@dmocko Thanks a lot.
